mobile/src/containers/GameSessionContainer.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from "react"
+import EncryptedStorage from "react-native-encrypted-storage"
+
+const GameSessionContainer = ({ children }) => {
+    const [gameCode, setGameCode] = useState(null)
+    const [teamId, setTeamId] = useState(null)
+    const [gameSession, setGameSession] = useState(null)
+    const [teamMember, setTeamMember] = useState(null)
+
+    useEffect(() => {
+        loadLocalGameSession().then((localGameSession) => {
+            if (localGameSession) {
+                setGameSession(localGameSession.gameSession)
+                setTeamId(localGameSession.teamId)
+                setTeamMember(localGameSession.teamMember)
+                setGameCode(localGameSession.gameCode)
```

Comment:
Same here, this is already part of `gameSession`, any reason we need a separate state for it?